src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObj = {};
+  const styles = sourceString.replaceAll(/[\r\n\s]+/g, '').split(';');
```

Comment:
The use of `replaceAll` with a regular expression to remove whitespace and newlines is correct. However, ensure that the input string is not empty before proceeding to split it, as this could lead to an array with an empty string element.